keyboards/pixelspace/shadow80/readme.md

```diff
@@ -0,0 +1,44 @@
+# Shadow80
+
+![shadow80](https://imgur.com/XLjjdMzh.jpg)
+A 80% layout keyboard with 74 RGB.
+Before enabling the maximum brightness of RGB lights, please confirm whether the USB device power supply supports it.
+This PCB about two versions ： hotswap and  solder.
```

Comment:
As this `readme.md` mentions hotswap and solder versions of this keyboard, could directory structure be amended to match? i.e.
```
keyboards/
  pixelspace/
    shadow80/
      chconf.h
      halfconf.h
      mcuconf.h
      readme.md
      matrix_diagram.md
      solder/
        info.json
        rules.mk
      hotswap/
        info.json
        rules.mk
      keymaps/
        default/
           keymap.c 
        via/
           keymap.c 
           rules.mk 
```

Reply:
There is no difference in key layout between hotswap and solder versions of PCB,So considering these, I think that not separating matching will Better.Thank you very much for your help!!!Many problems can be solved.!!!